mathics_django/web/media/js/doc.js

```diff
@@ -61,7 +60,6 @@ function loadDoc(page) {
 
 function showDoc() {
         const docLink = document.getElementById('doclink');
-        loadDoc('');
```

Comment:
The problem with reloading the documentation page here is that it breaks the search bar.